src/graphics/font.cpp

```diff
@@ -215,7 +215,7 @@ glm::vec2 Font::PreparedFontString::getUsedAreaSize() const
     float max_y = 0.0f;
     for(auto& d : data) {
         max_x = std::max(max_x, d.position.x);
-        max_y = std::max(max_y, d.position.y);
+        max_y = std::max(max_y, (float(getLineCount()) + 0.3f) * d.size);
```

Comment:
This might work but isn't correct, it's also calling getLineCount() for each character in the string, so that's quite inefficient.

Reply:
Yeah, sorry, I hadn't read getLineCount() and didn't think this through enough once I saw it working. I've got what I think is a much more correct solution written up and working now. I'll do some more extensive tests and screenshots later this evening.